airflow/models/taskinstance.py

```diff
@@ -3196,19 +3269,30 @@ def schedule_downstream_tasks(self, session: Session = NEW_SESSION, max_tis_per_
                     schedulable_ti.task = task.dag.get_task(schedulable_ti.task_id)
 
             num = dag_run.schedule_tis(schedulable_tis, session=session, max_tis_per_query=max_tis_per_query)
-            self.log.info("%d downstream tasks scheduled from follow-on schedule check", num)
+            ti.log.info("%d downstream tasks scheduled from follow-on schedule check", num)
```

Comment:
Would not be better to make it a `classmethod` and use `cls.log`?

Reply:
Good idea